dateparser_scripts/order_languages.py

```diff
@@ -92,6 +92,23 @@ def _get_language_order(language_locale_dict):
     return language_order
 
 
+def create_language_maps(language_order):
+    language_order.sort()
```

Comment:
this is quite minor, but it's better to not modify the arguments if the function is not explicitly called to modify them - and in this case, the function returns a value, so the called would not expect it to also sort the input.

Reply:
The sorting is required for the language mapping to work as this sorted list is used to group languages.

Reply:
@gavishpoddar sure, but you can still sort without affecting the input argument, e.g. using ``sorted`` function.

Reply:
@lopuhin, Thanks